src/components/member-data/member-data.tsx

```diff
@@ -1,10 +1,18 @@
 import { $, component$, useComputed$ } from "@builder.io/qwik";
-import type { MemberDataStore } from "~/routes/start/[...id]";
+import type { MemberDataStore, BillStore } from "~/routes/start/[...id]";
 import Button from "../button/button";
 import { sum } from "~/utils/math";
 
 export default component$(
-  ({ store, number }: { store: MemberDataStore; number: number }) => {
+  ({
+    store,
+    billStore,
+    number,
+  }: {
+    store: MemberDataStore;
+    billStore: BillStore;
```

Comment:
I'd rather not pass the entire store here as it's an extra dependency. What do you think about passing only a callback as prop?

Reply:
Yes, I would say it's a cleaner solution